docs/source/upcoming_release_notes/734-twincat-homing.rst

```diff
@@ -0,0 +1,34 @@
+7#34 twincat-homing
+#################
+
+API Changes
+-----------
+- Update twincat motors to use the correct homing PV.
+  This is an alternative PV to the normal motor record PVs for IOC/PLC
+  management reasons.
+  It is possible that this will break devices that have not updated to the
```

Comment:
Is it worth quantifying how many of these there are, or which need updating, etc.?

Reply:
I'm not sure how to quantify this. I'm not aware of any specifics and I've updated FEE, etc.

Reply:
Would checking the library summary for all deployed IOCs be a reasonable thing to do? https://pcdshub.github.io/lcls-plc-lfe-motion/v1.2.1/plc-lfe-motion_lfe_motion_summary.html#libraries

This could be something that gets added to our plc-summary page - it seems important enough to me, at least.

Reply:
That seems reasonable to me. I'll check the summary page for all the plcs that I know about in the afternoon.

Reply:
Checked all known repos for L2SI, only txi optics was far enough behind to be a concern

Reply:
Great!